.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E111,E114,E501,E121
+ignore = E111,E114,E501,E121,E241,E999
```

Comment:
What are these new exceptions for?

Reply:
multiple spaces after comma (for the giant list) and invalid syntax (since flake8 doesn't know about some python3 f-string stuff).

Reply:
Oh I see.  fstrings are python 3.6 and most likely out version of flake8 is too old   Is it a good idea to depend on such a recent version of python

More generally it seems like flake8 is not configurable enough for us we probably want to look for another such tool.

Can you add comments to the .flake8 file?

Reply:
I removed the f-strings, so now any python3 should work and I could remove the E999. Comments added to .flake8.